fastf1/api.py

```diff
@@ -1291,6 +1292,78 @@ def session_status_data(path, response=None, livedata=None):
     return data
 
 
+@Cache.api_request_wrapper
+def race_control_messages(path, response=None, livedata=None):
+    """Fetch and parse race control messages.
+
+    Race control messages are sent by race control to all teams to notify of
+    decisions and statuses of the session
+
+    Every message has the following attributes:
+
+        - Utc: Message timestamp
+        - Category (str): Type of message, "Other", "Flag", "Drs", "CarEvent"
+        - Message (str): Content of message
+
+    Other possible attributes are:
+
+        - Status (str): Status of context, e.g. "DISABLED" for disabling DRS
+        - Flag (str): Type of flag being waved "GREEN", "RED", "YELLOW",
+          "CLEAR", "CHEQUERED"
+        - Scope (str): Scope of message "Track", "Sector", "Driver"
+        - Sector (int): Affected track sector for sector-scoped messages
+        - RacingNumber (str): Affected driver for CarEvent messages
+
+    Args:
+        path (str): api path base string (usually ``Session.api_path``)
+        response: Response as returned by :func:`fetch_page` can be passed if
+        it was downloaded already.
```

Comment:
I need some minor changes here. The second line (and following lines) for any argument need to be indented with four spaces. Else the list of arguments is not rendered correctly.
You can also download the documentation build artifact from the CI run yourself and open the api.html file with any browser. Then you can check if everything looks correct.

Reply:
Changes made (ran against sphinx locally):
![Screenshot 2022-05-01 045246](https://user-images.githubusercontent.com/984973/166114920-49260cb9-df2e-401e-a895-003e7a87caa5.png)

Apologies for issues - it's the first time I've coded Python in anger.

Reply:
No worries, I just hope that my request for changes was not the source of your anger ;)

Reply:
Haha, I've just realised that I've used a phrase that may not have translated - doing something "in anger" in this context means "doing something seriously". I've only touched Python in a small way (mostly some scripts that work with this library), but this was the first time I've done some actual serious coding for the benefit of others :)

Reply:
I've never heard that phrase before. But that makes more sense now 😁